frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -761,15 +784,73 @@ export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProp
   );
 });
 
+const InitializationResourceAlert: React.SFC<InitializationResourceAlertProps> = (props) => {
+  const { initializationResource, csv } = props;
+
+  const initializationResourceKind = initializationResource?.kind;
+  const initializationResourceDefinition = props.csv.spec?.customresourcedefinitions?.owned?.find(
+    (api) => api.kind === initializationResourceKind,
+  );
+  const initializationResourceModelName = initializationResourceDefinition?.name;
+  const { group: initializationResourceGroup } = groupVersionFor(
+    initializationResource?.apiVersion,
+  );
+  const { findModel } = useModelFinder();
+  const model = findModel(
+    initializationResourceGroup,
+    initializationResourceModelName.split('.').shift(),
+  );
+  // Check if the CR is already present - only checks for the specific name/ns in the initialization-resource
+  const [customResource, customResourceLoaded] = useK8sGet<K8sResourceCommon>(
+    model,
+    initializationResource?.metadata.name,
+    initializationResource?.metadata.namespace || csv.metadata?.namespace,
```

Comment:
You should check if it's a namespaced resource in `model`. You only need a namespace if it's a namespaced resource.